src/components/shared/form_fields/AmountField.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 	<fieldset class="form-field form-field-amount" :class="`locale-${ $i18n.locale }`">
-		<div v-if="minimumAmountMessage!=''" class="minimum-message">
+		<div v-if="props.minimumAmountMessage!=''" class="minimum-message">
 			{{ props.minimumAmountMessage }}
```

Comment:
props are automatically injected into the template, no need to prefix them with `props`